### ISSUE TITLE
Can't connect to cluster on v0.4.1

### ISSUE THREAD

**Issue by ananace** [bitnami-labs/cabin#77]
Using token authorization, with HTTPS connection.

The application just shows the cluster as down, while causing the following errors to occur in the api server log;
```
nov 17 13:13:46 kbmaster01 kube-apiserver[2102]: I1117 13:13:46.162080    2102 logs.go:41] http: TLS handshake error from 10.253.130.53:44340: remote error: tls: unknown certificate
nov 17 13:13:46 kbmaster01 kube-apiserver[2102]: I1117 13:13:46.184796    2102 logs.go:41] http: TLS handshake error from 10.253.130.53:44341: remote error: tls: unknown certificate
nov 17 13:13:46 kbmaster01 kube-apiserver[2102]: I1117 13:13:46.192063    2102 logs.go:41] http: TLS handshake error from 10.253.130.53:44342: remote error: tls: unknown certificate
nov 17 13:13:46 kbmaster01 kube-apiserver[2102]: I1117 13:13:46.200071    2102 logs.go:41] http: TLS handshake error from 10.253.130.53:44343: remote error: tls: unknown certificate
nov 17 13:13:46 kbmaster01 kube-apiserver[2102]: I1117 13:13:46.216792    2102 logs.go:41] http: TLS handshake error from 10.253.130.53:44344: remote error: tls: unknown certificate
nov 17 13:13:46 kbmaster01 kube-apiserver[2102]: I1117 13:13:46.283266    2102 logs.go:41] http: TLS handshake error from 10.253.130.53:44345: remote error: tls: unknown certificate
```

I'm assuming that the application is not accepting the Kubernetes API certificate?

**Comment by jordanwilson230** [comment]
I've struggled getting Cabin connected with my cluster for way too long. I've tried each authentication method, changed various settings in GKE for cluster connections, etc. Importing works fine, but as all the other issues point out, it shows the cluster as down and lists no resources. I was so excited when I came across an article talking about Cabin...but this needs fixing fast.

**Comment by valentin2105** [comment]
Same problem here with k8s v1.8.6 Cluster. 
I use CFSSL self-signed certs. 

I think an  `insecure` box would be awesome !

**Comment by sebgoa** [comment]
Sorry for the long delay answering, we are in a transition period in terms of maintenance.

So the project is not abandoned, we just need to figure out how to move forward to solve these bugs. 

Of course we appreciate any PRs.

**Comment by Bhullnatik** [comment]
Hey everyone,

We just released version 0.5.0 that should fix this issue.
Please update your app, and let us know if it works!

**Comment by jordanwilson230** [comment]
@Bhullnatik After upgrading, I'm able to connect and see my clusters but again, no info is presented in any tab and namespace are not shown.
